release/src/Main.hs

```diff
@@ -40,6 +40,13 @@ import qualified SdkVersion
 scalaVersions :: [T.Text]
 scalaVersions = ["2.12.12", "2.13.3"]
 
+-- Hack: the logic that looks for missing transitive dependencies is broken, as it checks all build-time dependencies with `bazel query`
```

Comment:
Do we have a GH/KVL issue to keep track of this? If not, let's create one.

Reply:
Created: https://github.com/digital-asset/daml/issues/9195